internal/internal_workflow_testsuite.go

```diff
@@ -2232,6 +2232,10 @@ func (env *testWorkflowEnvironmentImpl) setLastCompletionResult(result interface
 	env.workflowInfo.lastCompletionResult = data
 }
 
+func (env *testWorkflowEnvironmentImpl) setLastError(failure error) {
```

Comment:
```suggestion
func (env *testWorkflowEnvironmentImpl) setLastError(err error) {
```
`err` is a common name for variables of `error` type.

Reply:
Thx, fixed.